x-pack/solutions/observability/plugins/streams/server/lib/streams/helpers/lifecycle.test.ts

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { WiredStreamDefinition } from '@kbn/streams-schema';
+import { findInheritedLifecycle, findInheritingStreams } from './lifecycle';
+
+describe(__filename, () => {
```

Comment:
Can we just name this `Lifecycle helpers` in plain language rather than a file name?